pkg/sql/parsers/tree/explain.go

```diff
@@ -58,6 +70,33 @@ func (node *ExplainStmt) Format(ctx *FmtCtx) {
 	}
 }
 
+//func (node *ExplainStmt) Format(ctx *FmtCtx) {
+//	ctx.WriteString("explain")
+//	format := node.explainImpl.Format
+//	if format != "" && format != "text" {
+//		ctx.WriteString(" format = ")
+//		ctx.WriteString(node.explainImpl.Format)
+//	}
+//	stmt := node.explainImpl.Statement
+//	switch stmt.(type) {
+//	case *ShowColumns:
+//		st := stmt.(*ShowColumns)
+//		if st.Table != nil {
+//			ctx.WriteByte(' ')
+//			st.Table.ToTableName().Format(ctx)
+//		}
+//		if st.ColName != nil {
+//			ctx.WriteByte(' ')
+//			st.ColName.Format(ctx)
+//		}
+//	default:
+//		if stmt != nil {
+//			ctx.WriteByte(' ')
+//			stmt.Format(ctx)
+//		}
+//	}
+//}
+
```

Comment:
I am not a big fan of comment out functions.  If not used, remove?   If it is for later, then put it next PR?

Reply:
Sorry, I'll delete the fmt printout and the comment